src/main/java/org/hyperskill/hstest/testing/execution/MainMethodExecutor.java

```diff
@@ -86,7 +91,8 @@ private void initByClassName(String className) {
             Class<?> clazz = Thread.currentThread().getContextClassLoader().loadClass(className);
             initByClassInstance(clazz);
         } catch (ClassNotFoundException | NoClassDefFoundError ex) {
-            initByNothing(className);
+            throw new ErrorWithFeedback("Cannot find a class with a main method.\n" +
```

Comment:
There's some code duplication going on. Could this be refactored in such a way that we need to throw `ErrorWithFeedback` just once in a single place?

Reply:
Left throwing an exception only in function `initByClassName`. It's ok if the error message for a package not found is the same as for a class not found?

Reply:
The feedback is correct in both cases. It references a package if you have searched in a specific package, or omits it if you have searched class everywhere. In both cases the main reason not to run user program - class with main method is not found, so that's why `Cannot find a class with a main method` should be in every variant of the feedback.